protocol/chainlib/common.go

```diff
@@ -18,10 +18,15 @@ const (
 )
 
 type parsedMessage struct {
-	serviceApi     *spectypes.ServiceApi
-	apiInterface   *spectypes.ApiInterface
-	requestedBlock int64
-	msg            interface{}
+	serviceApi       *spectypes.ServiceApi
+	apiInterface     *spectypes.ApiInterface
+	averageBlockTime int64
```

Comment:
remove, we don;t need to carry it in all of the messages especially when it's the same and chainParser is available